src/me/reducers/index.test.ts

```diff
@@ -0,0 +1,17 @@
+// Reducer
+import reducer from 'src/me/reducers'
+
+// Actions
+import {setMe} from 'src/me/actions/creators'
+
+// Mocks
+import {me} from 'src/me/mockUserData'
+
+describe('me reducer', () => {
+  it('can set me', () => {
+    const expected = me
+    const actual = reducer(undefined, setMe(expected))
```

Comment:
shouldn't this be `me` getting passed in and not `expected`?

Reply:
It doesn't make a difference in this particular case but I like that better too.  That way both expected and actual start out with the data form the import something is done to them (even though in expected's case it's very simple) and compared at the end.

Changed!